sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -5,12 +5,14 @@
 #### Features Added
 
 #### Breaking Changes
+* Modified `proactiveInit` key name to `proactiveInitCfg` in the diagnostic string. See [PR 36711](https://github.com/Azure/azure-sdk-for-java/pull/36711)
```

Comment:
This is !!!NOT!!! a breaking change. Diagnostics string is an opaque string - any customer parsing it, will see "breaks" even in hotfixes. The API intentionally just returns a String - there is  not even a guarantee that it will be json all the time - defintiely not any guarantee of not changing the json structure. Please remove this section.

Reply:
Fixed in next iteration.